server/db/migrate/0004_repo_owner_postgres.down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE repos DROP COLUMN user_id;
```

Comment:
same comment from the other day: I would leave the down.sql files empty...

Reply:
if not, rolling back and then migrating again won't succeed :/

Reply:
you can fix that by using `if [not] exists` in the `up.sql`:

```sql
ALTER TABLE if exists foo add if not exists bar type;
CREATE TABLE if not exists foo;
```

Reply:
Sqlite doesn't support that

Reply:
ah, can probably do the same with `begin` blocks with `exception` handling

in any case, I don't see why people would update and rollback and then update... whats the use case for that?